### PR TITLE
New operator sdk

### PR DESCRIPTION
**Pull request by e0ne**
This PR is based on #206 and fixes deployment issues.

### CODE REVIEW

**Comment by moshe010** [comment]
@e0ne  what is left to move this PR from WIP?

**Comment by e0ne** [comment]
@moshe010 apart of rebasing to remove temporary commit, I need to resolve following issues:
* fix rest of CI failures 
* fix 'make bundle' target - it doesn't work if 'operator-sdk' binary is not installed on the system

**Comment by moshe010** [comment]
@e0ne  can we remove the WIP? is the PR ready for review?

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
+        #image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```
Comment (moshe010) [inline config/default/manager_auth_proxy_patch.yaml:13]: why this comment?

Comment (e0ne) [reply config/default/manager_auth_proxy_patch.yaml]: Image is updated to v0.8.0 in the latest commit

**Comment by adrianchiris** [comment]
/LGTM, waiting the CI to be green

**Review by moshe010 — APPROVED** [review]

**Pull request merged**